### DISCUSSION TITLE
Modelling heat source

### DISCUSSION THREAD

**Discussion by ghost** [ansys/pyadditive#286] [Q&A] [answered]
@pkrull-ansys How does ADDITIVE models the laser heat source input? Is it a volumetric heat source or a flux on the upper surface? Is it a Gaussian shape or a Goldak model? Thanks!

**Comment by pkrull-ansys** [comment, marked as answer]
@maxcarrat90 Do you have access to https://ansyshelp.ansys.com/account/secured?returnurl=/Views/Secured/corp/v241/en/add_print/add_print_thermal_solver.html?

**Comment by ghost** [comment]
Yes, I do. Thank you for pointing it out!!
______________________________

Massimo Carraturo, Ph.D.
______________________________

University of Pavia
Department of Civil Engineering and Architecture
Structural Mechanics Division

Address: via Ferrata 3, 27100, Pavia, ITALY
E-mail:  ***@***.***
***@***.***>
Skype: maxcarrat
Phone:  (+39) 3393282346


Il giorno lun 4 mar 2024 alle ore 16:29 Peter Krull - ANSYS <
***@***.***> ha scritto:
